docs/tutorials/03-publishing-pilets.md

```diff
@@ -10,7 +10,7 @@ level: Elementary
 A working Piral system requires the following components:
 
 1. A functional frontend (referred to as "application shell" or "Piral instance")
-2. A backend / service delivering the pilets (referred to as "feed service")
+2. A backend / service delivering the pilets (referred to as **Feed Service**)
```

Comment:
This is a little bit inconsistent. At 1) we did the references with double quotes, now its in bold.

Reply:
Changed to double quotes to be consistent.